src/pgzero/screen.py

```diff
@@ -25,6 +31,63 @@ def make_color(arg):
     return tuple(pygame.Color(arg))
 
 
+def _get_platform_screenshot_path():
+    r"""Get the screenshot directory for pgzero.
+    Under Windows, this is %USERPROFILE%\Pictures\pgzero.
+    Under Linux/MacOS, it's ~/Pictures/pgzero.
+    If the platform is unsupported, it defaults to the CWD."""
+    if platform == "win32":
+        try:
+            home = os.environ["USERPROFILE"]
+        except KeyError:
+            raise KeyError("Couldn't find the user home directory for "
+                           "screenshots. Please set the %USERPROFILE% "
+                           "environment variable.")
+        return os.path.join(home, "Pictures", "pgzero")
+    elif platform in ["linux", "linux2", "darwin"]:
+        return os.path.expanduser(os.path.join("~", "Pictures", "pgzero"))
+    else:
+        print("WARNING: Device platform {} not recognized, thus no user folder"
+              " found. Falling back to current directory to save screenshots."
+              .format(platform))
```

Comment:
```suggestion
        print("WARNING: Device platform {} not recognized, thus no user folder"
              " found. Falling back to current directory to save screenshots."
              .format(platform),
              file=sys.stderr
        )
```

FWIW you can use f-strings and pathlib. Pygame Zero dates from before f-strings and pathlib but any versions of Python without them are long since end-of-life.

Reply:
Fixed.